docs/elasticsearch-spec.asciidoc

```diff
@@ -0,0 +1,103 @@
+[id="{p}-elasticsearch-specification"]
+== Elasticsearch Specification
+
+There are a number of settings which need to be considered before going into production related to Elasticsearch but also to Kubernetes.
```

Comment:
I think this sentence sounds a bit strange, @alaudazzi what do you think?

Reply:
I agree it's not great. I tried to write a draft of introduction.
This comes from https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html :)
We agreed with @alaudazzi that she will make a global pass when we have covered all the settings.